app/assets/stylesheets/provision.scss

```diff
@@ -0,0 +1,3 @@
+.row-button {
+  margin-bottom:15px;
```

Comment:
I think this is fine, but I'd go for `auto_provision.scss` or similar to the ERB which uses it.

Also you are missing end of the line (EOF).